test/parallel/test-icu-data-dir.js

```diff
@@ -7,8 +7,8 @@ const assert = require('assert');
 const { spawnSync } = require('child_process');
 
 const expected =
-    'could not initialize ICU (check NODE_ICU_DATA or ' +
-    '--icu-data-dir parameters)' + (common.isWindows ? '\r\n' : '\n');
+    `could not initialize ICU (check NODE_ICU_DATA or ` +
+    `--icu-data-dir parameters)${common.isWindows ? '\r\n' : '\n'}`;
```

Comment:
That's actually my fault, we had to fast-track a fix for https://github.com/nodejs/node/pull/13940 in https://github.com/nodejs/node/pull/13987.